agent/containers/images/visualizers/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Run collector (if needed), grafana setup, and grafana in parallel as grafana/collector are continuous processes
+# and the grafana setup needs to be run while grafana is up
+${COLLECTOR} &
+python3 grafana_setup.py &
+exec grafana-server                                         \
```

Comment:
Seems like we could fold this line into `grafana_setup.py`, along with the collector, and call it `run.py` instead.

We'd just have to `subprocess.run()` the `grafana-server` command first from the new `run.py` in the background, and then continue as normal.  We could also invoke the collector there too.

This would really tighten up this code.